spec/PeriodicBackgroundSync-index.bs

```diff
@@ -157,3 +159,51 @@ The <code><dfn method for=PeriodicSyncManager title="getTags()">getTags()</dfn><
 1. Let <var>currentTags</var> be a new [=sequence=].
 1. For each <var>registration</var> in |serviceWorkerRegistration|'s [=list of periodicsync registrations=], add |registration|'s [=tag=] to |currentTags|.
 1. [=Resolve=] |promise| with |currentTags|.
+
+## The <dfn>periodicSync event</dfn> ## {#periodicSync-event}
+<script type="idl">
+dictionary PeriodicSyncEventInit : ExtendableEventInit {
+    required DOMString tag;
+};
+
+[
+    Constructor(DOMString type, PeriodicSyncEventInit init),
+    Exposed=ServiceWorker
+] interface PeriodicSyncEvent : ExtendableEvent {
+    readonly attribute DOMString tag;
+  };
+</script>
+
+The {{PeriodicSyncEvent}} interface represents a [=firing=] [=periodicsync registration=].
+
+### Firing periodicSync events ### {#firing-periodicsync-events}
+Whenever the user agent changes to [=online=], it SHOULD [=fire a periodicSync event=] for each [=periodicsync registration=] whose [=registration state=] is [=pending=] and [=time to fire=] is now or in the past. The events MAY be fired in any order.
```

Comment:
Why is it significant that the events may fire in any order?

If it is, please clarify. It's it not, just leave it undefined until we find a good reason.

Reply:
removed.